pkg/command/context.go

```diff
@@ -581,20 +606,73 @@ func createContextWithTanzuEndpoint() (context *configtypes.Context, err error)
 	}
 
 	sanitizedEndpoint := sanitizeEndpoint(endpoint)
+	idpType := getIDPType(sanitizedEndpoint)
+	endpoints := getEndpoints(sanitizedEndpoint, idpType)
+	fmt.Printf("Create context %s: idpType = %v, endpoints: %+v\n", ctxName, idpType, endpoints)
```

Comment:
nit: Can we removed this line be removed or is it intentional?

Reply:
removed